indico/modules/events/papers/operations.py

```diff
@@ -329,8 +329,13 @@ def create_comment(paper, text, visibility, user):
                     session.user)
 
 
-def update_comment(comment, text, visibility):
-    changes = comment.populate_from_dict({'text': text, 'visibility': visibility})
+def update_comment(comment, text=None, visibility=None):
+    changed = {}
+    if text:
+        changed['text'] = text
+    if visibility:
```

Comment:
I think `is not None` would be better for those checks (this one here would fail if we had a visibility type with a value of 0 for example)